packages/block-scripts/scripts/dev.js

```diff
@@ -99,9 +41,14 @@ const script = async () => {
     ],
   };
 
-  const compiler = webpack(webpackConfig);
-  const server = new WebpackDevServer(webpackDevServerConfig, compiler);
-  await server.start();
+  const buildWebpackConfig = await generateBuildWebpackConfig("development");
+  const devWebpackConfig = await generateDevWebpackConfig();
+
+  webpack(buildWebpackConfig, () => {});
+
+  const devCompiler = webpack(devWebpackConfig);
+  const devServer = new WebpackDevServer(webpackDevServerConfig, devCompiler);
```

Comment:
Since `yarn dev` is now useful for testing blocks in EAs, we could set `open` in `webpackDevServerConfig` to `false`, and let users open the browsers themselves if they want (rather than spawning windows they might not need).

Reply:
I removed `open` option from the dev script in https://github.com/blockprotocol/blockprotocol/pull/317/commits/50657d58fc9f3ad819461f3c2f99328b33aa51c1.

We still have it in two other packages? Shall I get rid of it too for consistency? Not a fan of it TBH.

<img width="349" alt="Screenshot 2022-06-02 at 18 44 21" src="https://user-images.githubusercontent.com/608862/171693854-34dd0c15-f713-41c7-9cad-4c6ac29de347.png">

Reply:
Don't mind if you want to get rid of it in those.

I think it's different in that currently the only point in running `yarn dev` in the other two is to then look at the thing in the browser, so it isn't the situation we have now with `block-template` where you may want to run `yarn dev` without visiting the endpoint. But I don't feel strongly about it.

Reply:
https://github.com/blockprotocol/blockprotocol/pull/317/commits/d9c3c800ff427156825374b6f692753ffbdf87b6